src/appkit.cs

```diff
@@ -21628,6 +21759,10 @@ interface NSWorkspace : NSWorkspaceAccessibilityExtensions {
 		[Async]
 		[Export ("openURLs:withApplicationAtURL:configuration:completionHandler:")]
 		void OpenUrls (NSUrl[] urls, NSUrl applicationUrl, NSWorkspaceOpenConfiguration configuration, [NullAllowed] Action<NSRunningApplication, NSError> completionHandler);
+
+		[Mac (10,16)]
+		[Export ("iconForContentType:")]
+		NSImage IconForContentType (UTType contentType);
```

Comment:
Unless this is an overload of an existing method I would use

```suggestion
		NSImage GetIcon (UTType contentType);
```